### PR TITLE
Go version 1.7.4 -> 1.7.5

### PR DESCRIPTION
**Pull request by martinseener**
###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by martinseener** [comment]
Wondering why only appveyor tested this and not circleci dispite changing things there too!?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -69,16 +69,16 @@ second to sign our CLA, which can be found
 
 Installing Go
 -------------
-InfluxDB requires Go 1.7.4.
+InfluxDB requires Go 1.7.5.
```
Comment (phemmer) [inline CONTRIBUTING.md]: I know the wording pre-dates this change, but perhaps we should change it to "recommends" instead.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 ## v1.2.1 [unreleased]
 
+### Features
+
+- [#7926](https://github.com/influxdata/influxdb/pulls/7926): Go version 1.7.4 -> 1.7.5
```
Comment (phemmer) [inline CHANGELOG.md]: Bad url. It's "pull", not "pulls"

File: CONTRIBUTING.md (hunk 1 above)
Comment (martinseener) [inline CONTRIBUTING.md]: Sure! Will change the "requires"

File: CHANGELOG.md (hunk 2 above)
Comment (martinseener) [inline CHANGELOG.md]: Sorry, my fault.

**Comment by martinseener** [comment]
@phemmer changed both things and rebased it. Tests are running. Can you tell me why the CircleCI Tests do not run here??

**Comment by phemmer** [comment]
Sorry, didn't mean to give the impression that i'm a maintainer. Just a user who mucks around in the source code a lot :-)
So no idea about the CircleCI thing, sorry.

**Comment by martinseener** [comment]
Haha no worries. We'll wait for a maintainer and see, what has to be changed prior to the merge ;-)

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 ## v1.2.1 [unreleased]
 
+### Features
+
+- [#7926](https://github.com/influxdata/influxdb/pull/7926): Go version 1.7.4 -> 1.7.5
```
Comment (jwilder) [inline CHANGELOG.md]: Can you move this to the 1.3.0 section?

**Comment by martinseener** [comment]
@jwilder i've rebased it and moved it to 1.3.0 changes. you can now merge it ;)

**Comment by jwilder** [comment]
Closing this in favor of #8327

**Pull request closed without merging**